akka-stream/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -109,6 +109,12 @@ ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.ActorRefSinkActor$
 ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.ActorRefSink")
 ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.ActorRefSinkActor")
 
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.javadsl.Source.actorPublisher")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.javadsl.Sink.actorSubscriber")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.Source.actorPublisher")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.Sink.actorSubscriber")
+
+
```

Comment:
I'd just stick  with the usual one space line separation between issue additions